packages/odyssey-design-tokens/src/color/palette.json

```diff
@@ -109,32 +109,6 @@
           "value": "#8d6e97"
         }
       }
-    },
-    "primary": {
```

Comment:
Curious, why rename this file to palette? We refer to color as color in our docs today. A nit for sure, but would be good to be consistent everywhere.

Reply:
@arnoldsandoval-okta I split the single color.json apart into `color/funcitonal.json` and `color/palette.json` when I was playing with making all the palette colors private. Although we didn't go that route (making them private) I still think the distinction makes sense. All this said, there will be no change to the resulting tokens or their names though, style dictionary lets us organize the json however we'd like as long as its included with the build.